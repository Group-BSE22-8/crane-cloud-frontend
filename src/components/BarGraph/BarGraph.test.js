import React from "react";
import { shallow } from "enzyme";
import BarGraph from "./";

describe("Test the bar graph component", () => {
  const barGraphData = [{ name: "1" }, { name: "2" }];
  const barGraphProps = {
    data: barGraphData,
    height: 0,
    width: 0,
    barSize: 0,
    width_percentage: 0,
    height_percentage: 0,
  };
  const BarGraphComponent = shallow(<BarGraph {...barGraphProps} />);
  it("checks if the bar graph component matches the snapshot", () => {
    expect(BarGraphComponent).toMatchSnapshot();
  });
});
