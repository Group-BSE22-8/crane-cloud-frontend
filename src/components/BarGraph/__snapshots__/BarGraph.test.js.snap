// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the bar graph component checks if the bar graph component matches the snapshot 1`] = `ShallowWrapper {}`;
